docs/about/authors.rst

```diff
@@ -41,7 +41,11 @@ The following developers have contributed to the PyMeasure package:
 | Michele Sardo
 | Steven Siegl
 | Benjamin Klebel-Knobloch
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
I don't think you need to modify this file since you are already in it in the master.
There are also left over of the merge rebase which make me think that something went wrong during rebase.